openapi-v2.yaml

```diff
@@ -213,21 +213,401 @@ definitions:
+        type: array
+        x-omitempty: true
+        items:
+          $ref: "#/definitions/ArraySchemaEntry"
+
+  ArraySchemaEntry:
+    type: object
+    description: key/value pair representing an array schema map entry
+    properties:
+      key:
+        type: string
+      value:
+        $ref: "#/definitions/ArraySchema"
+
+  NonEmptyDomainList:
+    type: object
+    description: object containing non empty domains
+    properties:
+      nonEmptyDomains:
+        description: Array's non empty domain
+        type: array
+        x-omitempty: true
+        items:
+          $ref: "#/definitions/NonEmptyDomain"
+
+  NonEmptyDomain:
+    type: object
+    description: object representing a non-empty domain
+    required:
+      - nonEmptyDomain
+      - isEmpty
+    properties:
+      nonEmptyDomain:
+        description: Non Empty Domain of array
+        $ref: "#/definitions/DomainArray"
+      isEmpty:
+        description: Is non-empty domain really empty?
+        type: boolean
+        example: False
+      sizes:
+        description: Number of elements in DomainArray for var length
+        type: array
+        items:
+          type: integer
+          format: uint64
+
+  ArrayType:
+    description: TileDB array type
+    type: string
+    enum:
+      # Dense array
+      - dense
+      # Sparse array
+      - sparse
+
+  Attribute:
+    description: Attribute of array
+    type: object
+    required:
+      - name
+      - type
+      - filterPipeline
+      - cellValNum
+    properties:
+      name:
+        description: Attribute name
+        type: string
+        example: attribute1
+      type:
+        $ref: "#/definitions/Datatype"
+      filterPipeline:
+        $ref: "#/definitions/FilterPipeline"
+      cellValNum:
+        description: Attribute number of values per cell
+        type: integer
+        example: 1
+      nullable:
+        description: Is attribute nullable
+        type: boolean
+        example: true
+      fillValueValidity:
+        description: Default validity fill value for nullable attributes
+        type: boolean
+        example: true
+      fillValue:
+        description: The default fill value
+        type: array
+        items:
+          type: integer
+          format: uint8
+
+  FilterType:
+    description: TileDB filter types
+    type: string
+    enum:
+      # No-op filter
+      - FILTER_NONE
+      # Gzip compressor
+      - FILTER_GZIP
+      # Zstandard compressor
+      - FILTER_ZSTD
+      # LZ4 compressor
+      - FILTER_LZ4
+      # Run-length encoding compressor
+      - FILTER_RLE
+      # Bzip2 compressor
+      - FILTER_BZIP2
+      # Double-delta compressor
+      - FILTER_DOUBLE_DELTA
+      # Bit width reduction filter.
+      - FILTER_BIT_WIDTH_REDUCTION
+      # Bitshuffle filter.
+      - FILTER_BITSHUFFLE
+      # Byteshuffle filter.
+      - FILTER_BYTESHUFFLE
+      # Positive-delta encoding filter.
+      - FILTER_POSITIVE_DELTA
```

Comment:
Please add webp and float scaling

Reply:
I think those are also missing:
    FILTER_CHECKSUM_MD5
    FILTER_CHECKSUM_SHA256
    FILTER_DICTIONARY